.github/workflows/linter-yaml.yml

```diff
@@ -33,16 +33,35 @@ jobs:
 
       - name: Install dependencies
         if: steps.changed_files.outputs.any_changed == 'true'
-        run: pip install yamllint
+        run: pip install yamllint strictyaml argparse
 
-      - name: Run linter (config.yml)
+      - name: Run linter (config.yml) test
+        if: steps.changed_files.outputs.any_changed == 'true' && always()
+        uses: prince-chrismc/action-yamllint@main
+        with:
+          config: linter/yamllint_rules.yml
+          path: ${{ env.CONFIG_FILES_PATH }}
```

Comment:
I'm not sure to understand, what's the benefit of developing/using a separate github action, when the alternative is a single line call `yamllint --config-file linter/yamllint_rules.yml -f parsable ${{ env.CONFIG_FILES_PATH }}` with the same parameters ?

Reply:
I wanted to use someone else's hard work instead of using my head...


Then I needed to fix the action. 😢 

I will probably switch back since the author didnt answer :/